dgraph/cmd/zero/zero.go

```diff
@@ -226,18 +228,23 @@ func (s *Server) SetMembershipState(state *pb.MembershipState) {
 	if state.Zeros == nil {
 		state.Zeros = make(map[uint64]*pb.Member)
 	}
+
 	if state.Groups == nil {
 		state.Groups = make(map[uint32]*pb.Group)
 	}
+
+	//todo check if these groups are all zero or if there is any alpha
```

Comment:
how do you mean? 
Group 0 - all zero
Group 1..n - all alphas.

Reply:
Yes this was for my understanding to see which connections are made. I will remove these